src/_pytest/compat.py

```diff
@@ -78,7 +78,7 @@ def num_mock_patch_args(function):
     )
 
 
-def getfuncargnames(function, is_method=False, cls=None):
+def getfuncargnames(function, *, name: str = "", is_method=False, cls=None):
```

Comment:
Only reason I made this optional is to avoid breaking plugin code which might use this function (it is private I know, but it is common for people to use internal API unfortunately) and not in the mood to fix all calls in our test suite.

If others feel diferently, I will be happy to make this mandatory tomorrow.